crates/json/Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "json"
 version = "0.0.0"
 authors = ["Estuary Technologies, Inc"]
-edition = "2018"
+edition = "2021"
```

Comment:
I like it... out of curiosity what feature are you using that necessitated this upgrade?

Reply:
Yep, the `format!("with {my_local_variable}")` syntax is new to 2021.